virtualization/windowscontainers/quick-start/lift-shift-to-containers.md

```diff
@@ -257,7 +257,7 @@ When validating if the application to be containerized works on Windows containe
 |---|---|---|---|
 |.NET Framework versions 2.X and 3.X|.NET Framework 4.x|.NET Framework 3.5|Windows Server Core|
 |.NET Framework 4.x versions|.NET Framework 4.x|Build your .NET Framework container image with Server or Windows images|Windows Server Core|
-|.NET 6 or 7|.NET 6 or 7 respectively|Build your .NET container image with Windows or Server base images|Windows Nano Server or Server Core|
+|.NET 6 or later|Matching .NET container image, for example .NET 9 for .NET 9|Build your .NET container image with Windows or Server base images|Windows Nano Server or Server Core|
```

Comment:
> for example .NET 9 for .NET 9

That phrasing doesn't pop for me. As a user, I'm not quite sure how I'd apply ".NET 9" to containers. Perhaps "9.0" is closer?

I'm not sure this whole table makes much sense. For example, recommending .NET Framework 4.x images for a 2.x/3.x app won't work. That will launch FOD on Windows client, right?

Reply:
I assume yes. I don't feel qualified to update this table more usefully than I have done. Do you or @michaelsimons want to take a shot?